javascript-conditionals/main.js

```diff
@@ -0,0 +1,44 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+const isUnderFive=number=>number<5;
+const isEven=number=>number%2===0;
+const startsWithJ=string=>string[0]==='J';
+const isOldEnoughToDrink=({name,age})=>age>=21;
+const isOldEnoughToDrive=({name,age})=>age>=16;
+const isOldEnoughToDrinkAndDrive=({name,age})=>false; // dont drink AND drive, drink THEN drive:)
+function categorizeAcidity(pH){
+if(pH===7)return 'neutral';
+else if(pH>0&&pH<7) return 'acid';
+else if(pH>7&&pH<14) return 'base';
+else return 'invalid pH level';
+}
```

Comment:
You should be including brackets when using `else..if`. This is super hard to read otherwise. It's okay when utilizing the shorthand, and everything is on one line, but otherwise you should be utilizing brackets. Look at the below example.

```
function categorizeAcidity(pH) {
  if (pH === 7) {
    return 'neutral';
  } else if (pH < 7 && pH >= 0) {
    return 'acid';
  } else if (pH > 7 && pH <= 14) {
    return 'base';
  } else if (pH < 0 || pH > 14) {
    return 'invalid pH level';
  }
}
```